docs/modules/proj4/README.md

```diff
@@ -36,27 +36,32 @@ There are an infinite number of possible coordinate systems; therefore strict sy
 
 There are thousands of named "EPSG" projections. This module only includes aliases for those in the section below by default. To use a different EPSG projection, you can use https://epsg.io. For example, https://epsg.io/4326 defines standard longitude-latitude coordinates and lists multiple projection strings. Choose one of the `OGC WKT`, `ESRI WKT`, or `PROJ.4` strings listed.
 
-The epsg.io website also has a public API, e.g., for WGS 84: `https://epsg.io/?q=4326&format=json`
+You can also use MapTiler Coordinates API: https://api.maptiler.com/coordinates/search/4326.json?key=YOUR_MAPTILER_KEY&exports=true
```

Comment:
For someone without a lot of context about this, it feels random for OpenJS / Open Visualization to promote one specific commercial, key-requiring API. 

Can we add a header `## APIs` and a sentence that mentions that the espg.io team has migrated their free API to MapTiler, and perhaps something about the special relation that MapTiler has with this service that makes it natural to call you out?